src/dot/mod.rs

```diff
@@ -78,6 +78,52 @@ where
         Self::with_attr_getters(graph, config, &|_, _| String::new(), &|_, _| String::new())
     }
 
+    /// Create a `Dot` that uses the given functions to generate edge and node attributes.
+    ///
+    /// NOTE: `Config::EdgeNoLabel` and `Config::NodeNoLabel` should be set if you intend to
+    /// generate your own `label` attributes.
+    /// The getter functions should return an attribute list as a String. For example, if you
+    /// want to calculate a `label` for a node, then return `"label = \"your label here\""`.
+    /// Each function should take as arguments the graph and that graph's `NodeRef`.
```

Comment:
```suggestion
    /// Each function should take as arguments the graph and that graph's `EdgeRef` or `NodeRef`, respectively.
```
Takes `EdgeRef` or `NodeRef` depending on whether the closure is the edge or node attribute getter, correct?

Reply:
Good catch!